semantic_search/query_tree.py

```diff
@@ -11,8 +11,6 @@ def popularity(query, position):
 def location_distance(query, position):
     return semantic_functions.location_distance(query, position)
 
-def to_query_string(query_tree):
```

Comment:
should replace with `to_queries` function to match manuscript.

```
def to_queries():
  return [node["query"] for node in query_tree]
```

Corresponding usage (Listing 7.16):
```
queries = to_queries(query_tree)
reviews_collection = engine.get_collection("reviews")
reviews_collection.search(query=queries)
```

Reply:
Done